src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -186,6 +188,16 @@ public void initialize() {
     gameService.addOnRankedMatchNotificationListener(this::onMatchmakerMessage);
     // Always load chat immediately so messages or joined channels don't need to be cached until we display them.
     getView(NavigationItem.CHAT);
+
+    setListenerToSetExtraDragBarIfMinimized();
+  }
+
+  private void setListenerToSetExtraDragBarIfMinimized() {
```

Comment:
better name please, add instead for set and maybe also just call it `listen` instead of addListener